frontend/src/container/Login.jsx

```diff
@@ -240,14 +240,14 @@ class Login extends React.Component {
           <CardBody formClass='loginpage__card__form'>
             <form onSubmit={this.handleClickSubmit}>
               <InputGroupText
-                parentClassName='loginpage__card__form__groupemail'
+                parentClassName='loginpage__card__form__groupelogin'
                 customClass='mb-3 mt-4'
-                icon='fa-envelope-open-o'
-                type='email'
-                placeHolder={props.t('Email Address')}
-                invalidMsg={props.t('Invalid email')}
+                icon='fa-at'
+                type='text'
```

Comment:
I don't think this is required. Keeping type email seems good to me.
If the user wants to use his username, since username shouldn't have any special characters, the email optimized keyboard in good.
If the user wants to use his email, email keyboard is better.

I'm talking about mobile. On desktop it shouldn't change a lot.

If the change is about email validation by the html, I think you can disabled it since we do out own email validation. Confirm we do our own, I'm not sure

Btw, please confirm taht username cannot have special chars

Reply:
Allowed chars for username is `azAZ09-_` so errors can appears with `type=email`

Reply:
why that ?
`-` and `_` are email allowed characters

Reply:
After verification, yes you're right, in fact the browser show me an error (email invalid) because the field must contain a `@`

In FIrefox :
![image](https://user-images.githubusercontent.com/31687635/80383339-bec74a80-88a3-11ea-829b-b0520db8a1bf.png)

Reply:
indeed, I'd say remove the html validation on this field and we'll use our custom one

Reply:
but keep "email" type because it's easier for mobile users